content/community/changelog/deployment/v2/breaking-changes/_index.en.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Breaking changes
+description: Overview of breaking changes introduced into deployment in v2.0.0
+---
+
+1. Rename of deployment object might lead to a downtime equal to the startup time of the application.
+2. Default number of replicas changed from 1 to 2. If your application cannot run multiple replicas you need to override this (deplyoment.replicaCount=1)
```

Comment:
where should it be overrided? Including the name of file and path relative to repository would make the job easier for the app developers.